atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -89,7 +89,7 @@ def run(self):
         # Only run if the build was successful
         if self.workflow.build_process_failed:
             self.log.info("Not running for failed build")
-            return None, []
+            return []
```

Comment:
Should we reset `self.workflow.builder.image_id` here just in case then?

Reply:
I can do that, but self.workflow.builder.image_id is by default None

Reply:
It could have been changed in any plugin

Reply:
ok will reset it